LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015 Voytek Solutions LTD
+Copyright (c) 2016
```

Comment:
Do we need to get our laywers to approve this? :)

Reply:
Haha I don't think so.